compiler/include/driver.h

```diff
@@ -106,6 +106,8 @@ extern const char* CHPL_WIDE_POINTERS;
 extern const char* CHPL_LLVM;
 extern const char* CHPL_AUX_FILESYS;
 extern const char* CHPL_UNWIND;
+extern const char* CHPL_RUNTIME_SUBDIR;
```

Comment:
Since these are globals, I don't think we should only set them for --llvm compiles. I think we should always set them. 

I think this would also allow us to remove the `--llvm` special casing in printchplenv and some of the extra code added in `runPrintChplEnv()`